src/App.tsx

```diff
@@ -1,25 +1,42 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api';
 
 const App: React.FC = () => {
-  const [
-    selectedUserId,
-    // setSelectedUserId,
-  ] = useState(0);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+
+  useEffect(() => {
+    (async () => {
+      const loadedTodos = await getTodos();
+
+      setTodos(loadedTodos);
+    })();
```

Comment:
I would prefer writing a handler for this function, it is hard read what's going on here.